Expensify/App
# fix menu placement is broken on mobile

### Details
Reverted accidentally deleted file. and refactor Code for IOS.

### Fixed Issues

Fixes #3633
Fixes #3636
fixes #3631

### Tests | QA Steps
1. Open any chat on Mobile.
2. open context menu on any message.
3. It should open correctly from bottom to top.

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/24370807/122476215-70af6200-cfe3-11eb-9a38-66476601f403.mp4


#### Mobile Web


https://user-images.githubusercontent.com/24370807/122443202-f79d1400-cfbc-11eb-922c-cee46082cc9b.mp4


#### Desktop


#### iOS

https://user-images.githubusercontent.com/24370807/122443467-3df27300-cfbd-11eb-81fc-731eb991617c.mp4
#### Android
